app/controllers/users_controller.rb

```diff
@@ -0,0 +1,51 @@
+class UsersController < ApplicationController
+  before_action :require_login, only: [ :index, :logout, :destroy ]
+  def index
+    @users = User.all
+  end
+
+  def new
+    @user = User.new
+  end
+  
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      SendMailJob.perform_now(@user.id)
+      SendEmployeeMailJob.set(wait: 2.minutes).perform_later(@user.id) 
```

Comment:
This kind of logical thing should be written in the model file in a callback function. because if you create use from the console then this logic want work.